src/modules/temporary-file-check/dto/get-file-size-request.dto.ts

```diff
@@ -0,0 +1,24 @@
+import { EXAMPLES } from '@ukef/constants/examples.constant';
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
+
+export type GetFileSizeRequest = GetFileSizeRequestItem[];
+
+export class GetFileSizeRequestItem {
+  @ValidatedStringApiProperty({
+    description: 'The file name with the file extension.',
+    minLength: 1,
+    maxLength: 250,
+    pattern: /^[A-Za-z0-_.()\s]+$/,
+    example: EXAMPLES.FILE_NAME,
+  })
+  fileName: string;
+
+  @ValidatedStringApiProperty({
+    description: 'The location or file path in Azure storage from which the file needs to be loaded.',
+    minLength: 1,
+    maxLength: 250,
+    pattern: /^[A-Za-z/-_.()\s\-]+$/,
+    example: EXAMPLES.FILE_LOCATION_PATH,
+  })
+  fileLocationPath: string;
```

Comment:
I initially went off the `pattern`s in the ticket, but found that these didn't work for any of the real files in the dev storage account, so I adapted them to make them work. I meant to raise this when I put up this PR yesterday but didn't get the chance, sorry! I've seen that the ticket has since been updated (and I've also double checked these regexes against the ones in Mulesoft), so I've changed them to match Mulesoft

Reply:
Yep sorry for the ticket changing, when I made the comment above I realised I had put the wrong patterns on the ticket!